.github/scripts/run_tests.sh

```diff
@@ -92,17 +102,20 @@ fi
 export PYTORCH_TESTING_DEVICE_ONLY_FOR="xla"
 export CXX_ABI=$(python -c "import torch;print(int(torch._C._GLIBCXX_USE_CXX11_ABI))")
 
-if [[ -z "$RUN_BENCHMARK_TESTS" && -z "$RUN_CPP_TESTS1" && -z "$RUN_CPP_TESTS2" && -z "$RUN_PYTHON_TESTS" ]]; then
-  run_torch_xla_python_tests $PYTORCH_DIR $XLA_DIR $USE_COVERAGE
-  run_torch_xla_cpp_tests $PYTORCH_DIR $XLA_DIR $USE_COVERAGE
+if [[ -z "$RUN_BENCHMARK_TESTS" && -z "$RUN_CPP_TESTS1" && -z "$RUN_CPP_TESTS2" && -z "$RUN_PYTHON_TESTS" && -z "$RUN_TORCHBENCH_TESTS" ]]; then
+  run_torch_xla_python_tests $XLA_DIR $USE_COVERAGE
+  run_torch_xla_cpp_tests $XLA_DIR $USE_COVERAGE
   run_torch_xla_benchmark_tests $XLA_DIR
+  run_torch_xla_torchbench_tests $PYTORCH_DIR $XLA_DIR
 else
   # run tests separately.
   if [[ "$RUN_PYTHON_TESTS" == "python_tests" ]]; then
-    run_torch_xla_python_tests $PYTORCH_DIR $XLA_DIR $USE_COVERAGE
+    run_torch_xla_python_tests $XLA_DIR $USE_COVERAGE
   elif [[ "$RUN_BENCHMARK_TESTS" == "benchmark_tests" ]]; then
     run_torch_xla_benchmark_tests $XLA_DIR
+  elif [[ "$RUN_TORCHBENCH_TESTS" == "torchbench_tests" ]]; then
```

Comment:
Can you combine these with the `benchmark_tests`? That suite is very small and only takes ~1 minute to run.

Reply:
Combined